src/WebJobs.Script.WebHost/Management/FunctionsSyncManager.cs

```diff
@@ -99,27 +99,39 @@ public async Task<SyncTriggersResult> TrySyncTriggersAsync(bool checkHash = fals
                 await _syncSemaphore.WaitAsync();
 
                 var hashBlob = await GetHashBlobAsync();
-                if (checkHash && hashBlob == null)
+                if (isBackgroundSync && hashBlob == null)
                 {
-                    // short circuit before doing any work in cases where
-                    // we're asked to check/update hash but don't have
+                    // short circuit before doing any work in backgorund sync
```

Comment:
Do we need to add debug log in this case?

Reply:
It's not really an error case - people can run functions without a storage account configured. So I'd rather not have the noise logs.